packages/slice-machine/lib/models/common/widgets/ContentRelationship/Form.tsx

```diff
@@ -55,6 +56,16 @@ const WidgetForm = ({
         .filter((val): val is SelectValueType => val !== undefined)
     : null;
 
+  // removes deleted custom type from initial values on first render
+  useEffect(
```

Comment:
❓ As we filter the values i'm guessing before setting the field value for the first time, why isn't hitting the save button afterwards already enough to be able to save the custom type ?

Reply:
Since there's no change to the field value (no explicit interaction with it), setFieldValue in the onChange was never called before. I think because of this, the onSubmit of the form doesn't so anything